templates/mainapp/layout.html

```diff
@@ -349,7 +409,7 @@ <h4 class="modal-title" id="booktitle">Basic Modal</h4>
                     $('#booktitle').text(bookName);
                     $('#bookauthors').text(authors);
                     $('#bookimage').attr('src', imageurl);
-                    $("#bookrating").rateYo({
+                    $("#rateYo").rateYo({
                         rating: averagerating,
                         readOnly: true
```

Comment:
If it is read only, how is it that I can rate? Could you clarify?

Reply:
#rateYo is not the id of Rating Stars in Modal.

It is #rateYo-modal

Reply:
I understand that it was earlier there for the average rating, this function is for when the modal is opened from `More Details` Button, so you need to change this too, to `#rateYo-modal`

Reply:
So, i should change that and remove readOnly?

Reply:
Yes, confirm the behaviour, and let me know when it's ready for review